frontend_lib/src/container/PopupUploadFile.jsx

```diff
@@ -116,7 +116,9 @@ class PopupUploadFile extends React.Component {
       this.setState({
         fileUploadList: failedFileUploadList
       })
-      sendGlobalFlashMessage(props.t('Error while uploading file(s)'))
+      let message = props.t('Error while uploading file(s)')
+      if (failedFileUploadList.length === 1) message = failedFileUploadList[0].errorMessage
```

Comment:
Are the translations handled in this case?

Reply:
After an in depth lookup, it seem that the translation is indeed set and supposed to be set at higher level, so I suppose it is not specifically needed to be handled at this point.

Reply:
ok, cool